### ISSUE TITLE
NativeDeviceOrientationCommunicator().orientation(useSensor: true) receive NULL

### ISSUE THREAD

**Issue by plamprn** [rmtmckenzie/flutter_native_device_orientation#43]
When i updated to flutter 3 and use latest package version, function (useSensor: true) not already work!

i call this :
```
orientation = await NativeDeviceOrientationCommunicator().orientation(useSensor: true);
```

I receive this : 
```
E/MethodChannel#com.github.rmtmckenzie/flutter_native_device_orientation/orientation(18329): Failed to handle method call
E/MethodChannel#com.github.rmtmckenzie/flutter_native_device_orientation/orientation(18329): java.lang.NullPointerException: Attempt to invoke virtual method 'void com.github.rmtmckenzie.native_device_orientation.SensorOrientationReader.getOrientation(com.github.rmtmckenzie.native_device_orientation.IOrientationListener$OrientationCallback)' on a null object reference
E/MethodChannel#com.github.rmtmckenzie/flutter_native_device_orientation/orientation(18329): 	at com.github.rmtmckenzie.native_device_orientation.NativeDeviceOrientationPlugin$MethodCallHandler.onMethodCall(NativeDeviceOrientationPlugin.java:63)
E/MethodChannel#com.github.rmtmckenzie/flutter_native_device_orientation/orientation(18329): 	at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:262)
E/MethodChannel#com.github.rmtmckenzie/flutter_native_device_orientation/orientation(18329): 	at io.flutter.embedding.engine.dart.DartMessenger.invokeHandler(DartMessenger.java:295)
E/MethodChannel#com.github.rmtmckenzie/flutter_native_device_orientation/orientation(18329): 	at io.flutter.embedding.engine.dart.DartMessenger.lambda$dispatchMessageToQueue$0$DartMessenger(DartMessenger.java:319)
E/MethodChannel#com.github.rmtmckenzie/flutter_native_device_orientation/orientation(18329): 	at io.flutter.embedding.engine.dart.-$$Lambda$DartMessenger$AIEPqY6mWzaNK15HekX9bftoAXs.run(Unknown Source:12)
E/MethodChannel#com.github.rmtmckenzie/flutter_native_device_orientation/orientation(18329): 	at android.os.Handler.handleCallback(Handler.java:873)
E/MethodChannel#com.github.rmtmckenzie/flutter_native_device_orientation/orientation(18329): 	at android.os.Handler.dispatchMessage(Handler.java:99)
E/MethodChannel#com.github.rmtmckenzie/flutter_native_device_orientation/orientation(18329): 	at android.os.Looper.loop(Looper.java:214)
E/MethodChannel#com.github.rmtmckenzie/flutter_native_device_orientation/orientation(18329): 	at android.app.ActivityThread.main(ActivityThread.java:7073)
E/MethodChannel#com.github.rmtmckenzie/flutter_native_device_orientation/orientation(18329): 	at java.lang.reflect.Method.invoke(Native Method)
E/MethodChannel#com.github.rmtmckenzie/flutter_native_device_orientation/orientation(18329): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:494)
E/MethodChannel#com.github.rmtmckenzie/flutter_native_device_orientation/orientation(18329): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:965)
D/permissions_handler(18329): No permissions found in manifest for: []3
E/PLogger (18329): {PlatformException(error, Attempt to invoke virtual method 'void com.github.rmtmckenzie.native_device_orientation.SensorOrientationReader.getOrientation(com.github.rmtmckenzie.native_device_orientation.IOrientationListener$OrientationCallback)' on a null object reference, null, java.lang.NullPointerException: Attempt to invoke virtual method 'void com.github.rmtmckenzie.native_device_orientation.SensorOrientationReader.getOrientation(com.github.rmtmckenzie.native_device_orientation.IOrientationListener$OrientationCallback)' on a null object reference
E/PLogger (18329): 	at com.github.rmtmckenzie.native_device_orientation.NativeDeviceOrientationPlugin$MethodCallHandler.onMethodCall(NativeDeviceOrientationPlugin.java:63)
E/PLogger (18329): 	at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:262)
E/PLogger (18329): 	at io.flutter.embedding.engine.dart.DartMessenger.invokeHandler(DartMessenger.java:295)
E/PLogger (18329): 	at io.flutter.embedding.engine.dart.DartMessenger.lambda$dispatchMessageToQueue$0$DartMessenger(DartMessenger.java:319)
E/PLogger (18329): 	at io.flutter.embedding.engine.dart.-$$Lambda$DartMessenger$AIEPqY6mWzaNK15HekX9bftoAXs.run(Unknown Source:12)
E/PLogger (18329): 	at android.os.Handler.handleCallback(Handler.java:873)
E/PLogger (18329): 	at android.os.Handler.dispatchMessage(Handler.java:99)
E/PLogger (18329): 	at android.os.Looper.loop(Looper.java:214)
E/PLogger (18329): 	at android.app.ActivityThread.main(ActivityThread.java:7073)
E/PLogger (18329): 	at java.lang.reflect.Method.invoke(Native Method)
E/PLogger (18329): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:494)
E/PLogger (18329): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:965)
```
Flutter version : 
```
Flutter 3.0.0 • channel stable • https://github.com/flutter/flutter.git
Framework • revision ee4e09cce0 (13 days ago) • 2022-05-09 16:45:18 -0700
Engine • revision d1b9a6938a
Tools • Dart 2.17.0 • DevTools 2.12.2
```
And package version : 1.1.2

Thank you!!

**Comment by sidkris77** [comment]
Same issue here but with Flutter 2.10.5 and the plugin version 1.1.2. Had to downgrade to 1.0.0 to get it to work on Android (11 & 12) and iOS

`NativeDeviceOrientation orientation = await NativeDeviceOrientationCommunicator().orientation(useSensor: true);`

**Comment by plamprn** [comment]
> Same issue here but with Flutter 2.10.5 and the plugin version 1.1.2. Had to downgrade to 1.0.0 to get it to work on Android (11 & 12) and iOS
> 
> `NativeDeviceOrientation orientation = await NativeDeviceOrientationCommunicator().orientation(useSensor: true);`

I downgrade to 1.0.0 and 1.1.0 are get it to work on Android 12
Thank you

**Comment by rmtmckenzie** [comment]
Should be fixed in 1.1.3.
